tests/kafkatest/services/console_consumer.py

```diff
@@ -139,8 +178,8 @@ def start_node(self, node):
         super(ConsoleConsumer, self).start_node(node)
 
     def stop_node(self, node):
-        node.account.kill_process("java", allow_fail=False)
+        node.account.kill_process("java", allow_fail=True)
 
     def clean_node(self, node):
-        node.account.ssh("rm -rf /mnt/console_consumer.properties /mnt/consumer.log", allow_fail=False)
+        node.account.ssh("rm -rf %s" % ConsoleConsumer.PERSISTENT_ROOT, allow_fail=False)
```

Comment:
We should probably add a ducktape utility for cleaning up files. Will be useful to get that sort of stuff into utilities early if we ever want to try to support testing on Windows as well.